frontend_tests/node_tests/pm_list.js

```diff
@@ -94,6 +105,7 @@ test("build_private_messages_list", ({override}) => {
             is_active: false,
             url: "#narrow/pm-with/101,102-group",
             user_circle_class: undefined,
+            status_emoji_info: undefined,
```

Comment:
Do we want a test where `status_emoji_info` is not undefined to exercise the new code?